appshell/appshell_extensions.cpp

```diff
@@ -712,6 +712,12 @@ class ProcessMessageDelegate : public ClientHandler::ProcessMessageDelegate {
 
                 browser->GetHost()->SetZoomLevel(zoomLevel);
             }
+        }
+        else if (message_name == "InstallCommandLineTools") {
+            // Parameters:
+            //  0: int32 - callback id
```

Comment:
We could add a check for the number of arguments:
 `if (argList->GetSize() != 1) {error = ERR_INVALID_PARAMS}`